docs/api/components/schemas.yaml

```
@@ -3075,3 +3067,53 @@ doc_access_spans:
             entries:
               type: array
               description: Start sequence to end sequence. If the channel is currently granted, the end sequence will be zero.
+IndexSettings:
+  type: object
+  description: Settings for Global Secondary Indexes (GSI).
+  properties:
+    num_replicas:
+      description: This is the number of Global Secondary Indexes (GSI) to use for core indexes. Changing this number after index creation is initiated will not change the value.
+      type: number
+      default: 1
+      minimum: 0
+    num_partitions:
+      description: The number of partitions to use for the large indexes created by Sync Gateway. It is not recommended to set this unless you require additional horizontal scalability for individual indexes and have appropriately scaled your Query nodes to handle the increased query parallelism. If set, the recommended number is 8 and does not need to be directly related to the number of your Query nodes. Ensure documentation is read to understand the performance tradeoffs and instructions for migration if you have previously run with only one partition. See [/{db}/_index_init](#operation/post_db-_index_init) for more information.
+      type: number
+      default: 1
+IndexInitStatus:
+  description: The status of an asynchronous indexes initizational operation.
+  type: object
+  properties:
+    status:
+      description: The status of the current operation.
+      type: string
+      enum:
+        - running
+        - completed
+        - error
```

> stop: Stops tracking the indexes by Sync Gateway. These indexes will still be created on Couchbase Server.

Suggests "stopping" and "stopped" still need to be valid enum states for this?

```suggestion
        - running
        - completed
        - stopping
        - stopped
        - error
```